### PR TITLE
fix the single proces and the hwc spinner

### PR DESCRIPTION
**Pull request by Lhumd**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

[//]: # "Description of what you did.  If it is more complex, consider to add a"
[//]: # "'Summary' section."

Fix the single process.
Fix the HWC spinner using the frequency manager.

WARNING DURTY:
For the single process the control is activated only upon starting the graph.
Meaning the calibration can be done only when the graph is started.

## How I Tested

[//]: # "Explain how you tested your changes"

I ran it on Bolt.

## Do not merge before

[//]: # "Link other open pull request here that need to be merged first."
[//]: # "Remove this section if it does not apply."

- [x] https://github.com/machines-in-motion/real_time_tools/pull/28

## I fulfilled the following requirements

[//]: # "Please make sure you followed these steps before requesting a review."
[//]: # "Check the boxes in the list below, when done."

- [x] All new code is formatted according to our style guide (for C++ run clang-format, for Python, run flake8 and fix all warnings).
- [x] All new functions/classes are documented and existing documentation is updated according to changes.
- [x] No commented code from testing/debugging is kept (unless there is a good reason to keep it).

### CODE REVIEW

File: src/dynamic_graph_manager.cpp [hunk 1]
```diff
@@ -436,12 +436,19 @@ void DynamicGraphManager::run_hardware_communication_process()
...
 {
     initialize_dynamic_graph_process();
+    std::cout << "INI" << std::endl;
 
     initialize_hardware_communication_process();
+    std::cout << "Hard" << std::endl;
     get_ros_node(hw_com_ros_node_name_);
+    std::cout << "Ros node" << std::endl;
     ros_add_node_to_executor(hw_com_ros_node_name_);
+    std::cout << "ross add " << std::endl;
```
Comment (jviereck) [inline src/dynamic_graph_manager.cpp]: These look like debug messages. Do you want to keep the std out here or should they go away?

**Review by jviereck — APPROVED** [review]
Looks good. That said, maybe @MaximilienNaveau could have a look at it as well?

**Comment by MaximilienNaveau** [comment]
We co-wrote it with elham

File: src/dynamic_graph_manager.cpp (hunk 1 above)
Comment (Lhumd) [reply src/dynamic_graph_manager.cpp]: I deleted them. Thanks.

**Comment by jviereck** [comment]
Great, from my side, feel free to merge this.

**Review by MaximilienNaveau — APPROVED** [review]

**Pull request merged**